Classes/Form/Wizard/Select.php

```diff
@@ -32,9 +32,11 @@ class Select extends AbstractWizard {
 	protected $icon = 'list.gif';
 
 	/**
-	 * @var string
+	 * @var array
 	 */
-	protected $script = 'wizard_list.php';
+	protected $module = array(
+		'name' => 'wizard_list'
```

Comment:
This should probably be `'name' => 'wizard_select'` - that's the only thing I could spot in this PR :)